.github/workflows/boostrap-workflow.yml

```diff
@@ -0,0 +1,20 @@
+name: bootstrap workflow
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
+    - name: run boostrap.sh
+      run: |
+        cd ./
+        chmod +x bootstrap.sh
```

Comment:
A shell script permissions should be updated and tracked in git. Using `chmod` at this point make me think that it doesn't.